tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -2421,10 +2422,11 @@ def _test_sparse_to_dense(sparse_indices, sparse_values, default_value, output_s
         )
         oshape = tf.constant(output_shape, shape=output_shape.shape, dtype=str(output_shape.dtype))
 
+        # Output shape depends on a dynamic input, use VM.
         if default_value == None:
             output = tf.sparse_to_dense(indices, oshape, values)
             compare_tf_with_tvm(
-                [sparse_indices, sparse_values], ["indices:0", "values:0"], output.name
+                [sparse_indices, sparse_values], ["indices:0", "values:0"], output.name, mode="vm"
```

Comment:
This test was previously working on the graph runtime just because the dynamic input is bound to a constant tensor before `relay.build(...)` in TF tests. Such usage of dynamic inputs makes no sense so I just changed it to use vm for testing.